sql/core/src/test/scala/org/apache/spark/sql/GeneratorFunctionSuite.scala

```diff
@@ -344,12 +345,73 @@ class GeneratorFunctionSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("Supported nested inner generators") {
```

Comment:
If you don't mind, `SPARK-31049: ` prefix?

Reply:
Yea, of course, not.